internal/providersdkv2/resource_vault_cluster.go

```diff
@@ -232,6 +232,57 @@ If a project is not configured in the HCP Provider config block, the oldest proj
 							Optional:    true,
 							Sensitive:   true,
 						},
+						"http_basic_user": {
```

Comment:
Also not related to this line specifically, but commenting here to keep it threaded. We should probably wait until the public client changes from https://github.com/hashicorp/cloud-vault-service/pull/4004 for the `HashicorpCloudVault20201125HTTPEncodingCodec` type are updated in `hcp-sdk-go`.

There's an automated weekly release for `hcp-sdk-go` [here](https://github.com/hashicorp/hcp-sdk-go/releases) (seems like it happens on Tuesdays), which will trigger an automated Bump PR like [this one](https://github.com/hashicorp/terraform-provider-hcp/pull/674) in `terraform-provider-hcp`. We should wait until those are in before merging this PR. 

We can probably just wait for those to happen next week, but @jaireddjawed please check with @aidan-mundy or @aslamovamir (who I think needed to do something similar recently) if you need to expedite.

Reply:
👋 I don't think this has been resolved yet, since hcp-sdk-go didn't seem to pick up the public client changes on its latest commit: https://github.com/hashicorp/hcp-sdk-go/blob/3a6dbab598f6d08edda589c24840650b8761f795/clients/cloud-vault-service/stable/2020-11-25/models/hashicorp_cloud_vault20201125_http_encoding_codec.go#L31-L44 

checking with you and Amir in Slack how Amir did this recently.

Reply:
The public client change is on `main` of `hcp-sdk-go` but it's not part of the `0.74.0` release (see [here](https://github.com/hashicorp/hcp-sdk-go/compare/v0.74.0...main)), which is currently what the terraform provider is using (see [here](https://github.com/hashicorp/terraform-provider-hcp/blob/main/go.mod#L13)). We need these two things before we can merge this PR:
1. `0.75.0` release to `hcp-sdk-go` first
2. Bump to `0.75.0` of `hcp-sdk-go` in this repo's `go.mod`

Then we can merge this change.